int8 id
float32 present_radian
float32 present_velocity
float32 present_current
